dots_and_boxes/aiService.ts

```diff
@@ -0,0 +1,37 @@
+import {AiService, IMove} from '../common/common';
```

Comment:
Need some commenting in this file, there's no much code but there are some loops who needs explanation.